client/src/modules/stock/requisition/modals/status.tmpl.html

```diff
@@ -0,0 +1,54 @@
+<div class="modal-header">
+  <ol class="headercrumb">
+    <li class="static"><span translate>FORM.LABELS.STATUS</span> <span translate>REQUISITION.STOCK_REQUISITION</span></li>
+    <li class="title">{{ $ctrl.requisition.reference }}</li>
+  </ol>
+</div>
+
+<form 
+  name="ModalForm"
```

Comment:
Why does this modal exist?  Shouldn't these be modified automatically?  Except for canceled which is an action, not a status.

Reply:
The modals can also display a small message, to remind the user of the scope of each action
![image](https://user-images.githubusercontent.com/8035588/94549443-249c7180-024a-11eb-9308-f2ecedf6d126.png)

Reply:
Personally, I find having one radio button confusing. 

I think a better interface would use ModalAction for this - where the message is just displayed and the user chooses ok/cancel.  This would involve the user clicking "cancel", "confirm", etc on a dropdown menu in the 'actions' column first though.